### ISSUE TITLE
Replace lxxiASM with NormalASM

### ISSUE THREAD

**Issue by shrapnelnet** [xJon/The-1.12.2-Pack#965]
### Was this feature suggested before?

- [X] No, it was not.

### What is your suggestion?

I'd rather not mention the mod by name, but the developer and mod itself use a word that directly translates to pedophile in japanese. NormalASM is a direct drop in replacement and I have tested it and verified that it works on temurin version 8. 

Here is a link to the repository: https://github.com/mirrorcult/NormalASM

**Comment by xJon** [comment]
I don't believe loli translates to a podophile in Japanese, and also NormalASM is very outdated.
That being said, the mod is expecting an overhauling update soon that will include rebranding as well as a lot of other improvements.
